src/mui-components/MUITextField/MUITextField.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { TextField } from "@mui/material";
+import { Controller } from "react-hook-form";
+import { MUITextFieldProps } from "../../Constants/application.type";
+
+export default function MUITextField({
```

Comment:
Can we use one general component for the textfield?

Reply:
sure, will remove this file